tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/game_preferences/tgui.tsx

```diff
@@ -1,15 +1,22 @@
-import { CheckboxInput, FeatureToggle } from "../base";
+import { CheckboxInput, FeatureToggle } from '../base';
 
 export const tgui_fancy: FeatureToggle = {
-  name: "Enable fancy tgui",
-  category: "UI",
-  description: "Makes tgui windows look better, at the cost of compatibility.",
+  name: 'Enable fancy tgui',
```

Comment:
eh whatever

Reply:
sorry, just prettier doing its thing - i think @stylemistake set the single quotation flag